### PR TITLE
Update release notes for SDK 3.2.0

### PR DESCRIPTION
**Pull request by avsej**

### CODE REVIEW

**Review by maria-robobug — APPROVED** [review]
👍

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 1]
```diff
@@ -44,27 +43,116 @@ Couchbase SDK library:
...
+  Map query error code 13014 to `AuthenticationFailure` exception.
+
+* https://issues.couchbase.com/browse/RCBC-227[RCBC-227]:
+  Manage remote links for analytics
+
+* https://issues.couchbase.com/browse/RCBC-283[RCBC-283]:
+  Add collections support for search queries
+
+* https://issues.couchbase.com/browse/RCBC-311[RCBC-311]:
+  Fix scope qualifer encoding for analtyics
```
Comment (osfameron) [inline modules/project-docs/pages/sdk-release-notes.adoc]: typo

File: modules/project-docs/pages/sdk-release-notes.adoc [hunk 2]
```diff
@@ -44,27 +43,116 @@ Couchbase SDK library:
...
+ Implement tracing. This feature is enabled by default, but could be disabled in connection string with
+`enable_tracing=false` or programmatically:
++
+[source,ruby]
+----
+options = Cluster::ClusterOptions.new
+options.enable_tracing = false
+----
++
+Extra options:
```
Comment (osfameron) [inline modules/project-docs/pages/sdk-release-notes.adoc:101]: Nice example!

**Comment by osfameron** [comment]
Looks good to me 👍

**Review by RichardSmedley — APPROVED** [review]
:+1:

**Pull request merged**